packages/monorepo-builder/packages/release/src/ReleaseWorker/PushNextDevReleaseWorker.php

```diff
@@ -21,18 +23,28 @@ final class PushNextDevReleaseWorker implements ReleaseWorkerInterface
      */
     private $versionUtils;
 
-    public function __construct(ProcessRunner $processRunner, VersionUtils $versionUtils)
+    /**
+     * @var string
+     */
+    private $branchName;
+
+    public function __construct(
+        ProcessRunner $processRunner,
+        VersionUtils $versionUtils,
+        ParameterProvider $parameterProvider
+        )
     {
         $this->processRunner = $processRunner;
         $this->versionUtils = $versionUtils;
+        $this->branchName = $parameterProvider->provideStringParameter(Option::DEFAULT_BRANCH_NAME);
     }
 
     public function work(Version $version): void
     {
         $versionInString = $this->getVersionDev($version);
 
         $gitAddCommitCommand = sprintf(
-            'git add . && git commit --allow-empty -m "open %s" && git push origin master',
+            'git add . && git commit --allow-empty -m "open %s" && git push origin ' . $this->branchName,
```

Comment:
can use `%s` and add more parameter to sprintf

Reply:
https://github.com/symplify/symplify/pull/2959/commits/fca87770e693c0589fc2d53ef8a888e7a3d29d48